modules/manage/pages/manage-settings/general-settings.adoc

```diff
@@ -97,14 +97,18 @@ image::manage-settings/node-availability.png["The Node Availability panel",548,a
 
 The following checkboxes are provided:
 
-* *Auto-failover after _x_ seconds for up to _y_ node*: After the timeout period set here as _x_ seconds has elapsed, an unresponsive or malfunctioning node is failed over, provided that the limit on actionable events set here as _y_ has not yet been reached.
+* *Auto-failover after _x_ seconds for up to _y_ node*: After the timeout period set here as _x_ seconds has elapsed, an unresponsive or malfunctioning node is failed over, provided that the limit on actionable events set here as _y_ (with the default value of 1) has not yet been reached.
 Data replicas are promoted to active on other nodes, as appropriate.
 This feature can only be used when three or more nodes are present in the cluster.
 The number of seconds to elapse is configurable: the default is 120; the minimum permitted is 5; the maximum 3600.
 This option is selected by default.
+
 * *Auto-failover for sustained data disk read/write failures after _z_ seconds*: After the timeout period set here as _z_ seconds has elapsed, a node is failed over if it has experienced sustained data disk read/write failures.
-The timeout period is configurable: the default length is 120 seconds; the minimum permitted is 5; the maximum 3600.
+The timeout period is configurable: the default length is 240 seconds; the minimum permitted is 5; the maximum 3600.
```

Comment:
I'm not aware we changed it from 120 to 240 seconds, could you point me to the improvement?

Reply:
My error - I have corrected.